pandas/_libs/algos.pyi

```diff
@@ -92,58 +75,40 @@ def nancorr_kendall(
 #    uint16_t
 #    uint8_t
 
-
 def validate_limit(nobs: int | None, limit=None) -> int: ...
-
-
 def pad(
-    old: np.ndarray,   # ndarray[algos_t]
-    new: np.ndarray,   # ndarray[algos_t]
+    old: np.ndarray,  # ndarray[algos_t]
+    new: np.ndarray,  # ndarray[algos_t]
     limit=None,
 ) -> np.ndarray: ...  # np.ndarray[np.intp, ndim=1]
-
-
 def pad_inplace(
     values: np.ndarray,  # algos_t[:]
-    mask: np.ndarray,    # uint8_t[:]
+    mask: np.ndarray,  # uint8_t[:]
     limit=None,
 ) -> None: ...
-
-
 def pad_2d_inplace(
     values: np.ndarray,  # algos_t[:, :]
-    mask: np.ndarray,    # const uint8_t[:, :]
+    mask: np.ndarray,  # const uint8_t[:, :]
     limit=None,
-) -> None:
-    ...
-
-
+) -> None: ...
 def backfill(
     old: np.ndarray,  # ndarray[algos_t]
     new: np.ndarray,  # ndarray[algos_t]
     limit=None,
-) -> np.ndarray:  # np.ndarray[np.intp, ndim=1]
-    ...
-
+) -> np.ndarray: ...
 def backfill_inplace(
     values: np.ndarray,  # algos_t[:]
-    mask: np.ndarray,    # uint8_t[:]
+    mask: np.ndarray,  # uint8_t[:]
     limit=None,
 ) -> None: ...
-
-
 def backfill_2d_inplace(
     values: np.ndarray,  # algos_t[:, :]
-    mask: np.ndarray,    # const uint8_t[:, :]
+    mask: np.ndarray,  # const uint8_t[:, :]
     limit=None,
 ) -> None: ...
-
-
 def is_monotonic(
-    arr: np.ndarray,  # ndarray[algos_t, ndim=1]
```

Comment:
the idea with the current formatting is to have a single arg on the line so it is super-obvious which argument the comment refers to